src/main/java/ch/hevs/cloudio/endpoint/CloudioPersistence.java

```diff
@@ -0,0 +1,99 @@
+package ch.hevs.cloudio.endpoint;
+
+import java.util.Set;
+
+public interface CloudioPersistence {
+
+    /**
+     * Open the persistence database
+     */
+    void openDatabase();
```

Comment:
Better name would be just `open()`  and `close()` since there might be a memory based implementation.